modules/op-release-notes-1-11.adoc

```diff
@@ -14,12 +14,14 @@ In addition to fixes and stability improvements, the following sections highligh
 
 [NOTE]
 ====
-You can use {pipelines-title} on the {product-title} cluster that runs on ARM hardware. This update includes support for the `ClusterTask` resources where images are available and the Tekton CLI tool on ARM hardware.
+Before upgrading to the {pipelines-title} Operator 1.11, ensure that you have at least installed the {product-title} 4.12.19 or 4.13.1 version on your cluster.
 ====
 
 [id="pipelines-new-features-1-11_{context}"]
 === Pipelines
 
+* With this update, you can use {pipelines-title} on the {product-title} cluster that runs on ARM hardware. You have support for the `ClusterTask` resources where images are available and the Tekton CLI tool on ARM hardware.
```

Comment:
this is not part of the pipelines but the whole product.

Reply:
@piyush-garg Koustav asked me to move this to pipelines section today in the same thread: https://redhat-internal.slack.com/archives/CG5GV6CJD/p1687244275707269?thread_ts=1687242922.488079&cid=CG5GV6CJD